bindings/node/addon/mongocrypt.cc

```diff
@@ -18,113 +19,112 @@ struct InstanceData {
 };
 
 struct MongoCryptStatusDeleter {
-    void operator()(mongocrypt_status_t* status) {
+    void operator()(mongocrypt_status_t *status) {
         mongocrypt_status_destroy(status);
     }
 };
 
-Object ExtractStatus(Env env, mongocrypt_status_t* status) {
+Object ExtractStatus(Env env, mongocrypt_status_t *status) {
     Object result = Object::New(env);
     result["type"] = Number::New(env, mongocrypt_status_type(status));
     result["code"] = Number::New(env, mongocrypt_status_code(status));
-    const char* message = mongocrypt_status_message(status, nullptr);
+    const char *message = mongocrypt_status_message(status, nullptr);
     if (message != nullptr) {
         result["message"] = String::New(env, message);
     }
 
     return result;
 }
 
-std::unique_ptr<mongocrypt_binary_t, MongoCryptBinaryDeleter>
-BufferToBinary(Uint8Array node_buffer) {
-    uint8_t* buffer = node_buffer.Data();
+std::unique_ptr<mongocrypt_binary_t, MongoCryptBinaryDeleter> Uint8ArrayToBinary(Uint8Array node_buffer) {
+    uint8_t *buffer = node_buffer.Data();
     size_t buffer_len = node_buffer.ByteLength();
     return std::unique_ptr<mongocrypt_binary_t, MongoCryptBinaryDeleter>(
         mongocrypt_binary_new_from_data(buffer, buffer_len));
 }
 
-Uint8Array BufferFromBinary(Env env, mongocrypt_binary_t* binary) {
-    const uint8_t* data = mongocrypt_binary_data(binary);
+Uint8Array BufferFromBinary(Env env, mongocrypt_binary_t *binary) {
+    const uint8_t *data = mongocrypt_binary_data(binary);
     size_t len = mongocrypt_binary_len(binary);
     return Buffer<uint8_t>::Copy(env, data, len);
 }
 
-Uint8Array BufferWithLengthOf(Env env, mongocrypt_binary_t* binary) {
+Uint8Array BufferWithLengthOf(Env env, mongocrypt_binary_t *binary) {
     size_t len = mongocrypt_binary_len(binary);
     return Buffer<uint8_t>::New(env, len);
 }
 
-void CopyBufferData(mongocrypt_binary_t* out, Uint8Array buffer, size_t count) {
+Uint8Array Uint8ArrayFromValue(Napi::Value v, std::string argument_name) {
+    if (!v.IsTypedArray()) {
```

Comment:
```suggestion
    if (!v.IsTypedArray() || v.As<TypedArray>().TypedArrayType() != napi_uint8_array) {
```

(or otherwise I think you’re also fine just using `TypedArray` instead of `Uint8Array` everywhere)